### DISCUSSION TITLE
progress bar style

### DISCUSSION THREAD

**Discussion by internet-coffee** [z-huang/music#1396] [General]
I hope to change the progress bar style back to version 0.5.5,I think it looks better
![Screenshot_20240816_172545_InnerTune](https://github.com/user-attachments/assets/b18029e8-b6b6-43ef-900f-bb44c24dbaac)
![Screenshot_2024 08 16_17 24 55 959](https://github.com/user-attachments/assets/43af8bbf-d52d-4f74-b6ae-dfdca0a955c4)

**Comment by z-huang** [comment]
No. Material 3 has updated seek bar style to the current one.
